[MOSIP-43787] Added index on credential_event_store for status filtering from v1.2.1.2 upgrade scripts, rollback & upgrade from 1.3.0-beta.1 to 1.3.0.

NidhiKumari0201 opened the pull request:
[MOSIP-43787] Added index on credential_event_store for status filtering from v1.2.1.2 upgrade scripts, rollback & upgrade from 1.3.0-beta.1 to 1.3.0.

[MOSIP-43787]: https://mosip.atlassian.net/browse/MOSIP-43787?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

NidhiKumari0201 committed:
[MOSIP-43787] Added index on credential_event_store for status filtering from v1.2.1.2 upgrade scripts

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
added script required to rollback from 1.3.0-beta.1 to 1.3.0.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
added script required to rollback from 1.3.0-beta.1 to 1.3.0.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

ashok-ksharma commented on db_upgrade_scripts/mosip_ida/sql/1.2.1.2_to_1.3.0_rollback.sql:
1. Remove this line : https://github.com/NidhiKumari0201/id-authentication/blob/patch-1/db_upgrade_scripts/mosip_ida/sql/1.2.1.2_to_1.3.0_rollback.sql#L4
2. Line **"-- Below script required to rollback from 1.3.0-beta.2 to 1.3.0."** should be added above this because these are the new scripts : https://github.com/NidhiKumari0201/id-authentication/blob/patch-1/db_upgrade_scripts/mosip_ida/sql/1.2.1.2_to_1.3.0_rollback.sql#L45.
3. These lines are duplicated. Remove the same : https://github.com/NidhiKumari0201/id-authentication/blob/patch-1/db_upgrade_scripts/mosip_ida/sql/1.2.1.2_to_1.3.0_rollback.sql#L263-L265

NidhiKumari0201 replied:
done

ashok-ksharma commented on db_upgrade_scripts/mosip_ida/sql/1.2.1.2_to_1.3.0_upgrade.sql:
Follow the same comments as in rollback file.

NidhiKumari0201 replied:
done

NidhiKumari0201 committed:
Update upgrade script as per review changes

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
added Below script required to rollback from 1.3.0-beta.2 to 1.3.0.

Added rollback script for version 1.3.0-beta.2.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
removed -- Below script required to upgrade from 1.3.0-B2 to 1.3.0

Updated upgrade script to migrate Spring batch version to 5.0 and adjust table columns.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
removed -- Below script required to rollback from 1.3.0-beta.1 to 1.3.0.

Remove unnecessary comment and clean up rollback script.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
Clean up upgrade script by removing duplicate column addition

Removed redundant ALTER TABLE statement for ca_cert_type column.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
Remove duplicate column rollback script

Removed the ca_cert_type column rollback script from the SQL file.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>

NidhiKumari0201 committed:
added -- Below script required to upgrade from 1.3.0-beta.2 to 1.3.0.

Signed-off-by: Nidhi <nidhi.k@cyberpwn.com>
